lib/utils.js

```diff
@@ -294,12 +294,15 @@ api.computeCapabilityChain = async ({
  *
  * @return {Object} {valid, error}.
  */
-api.validateCapabilityChain = ({capability, capabilityChain}) => {
-  if(capabilityChain.length > constants.MAX_CHAIN_LENGTH) {
+api.validateCapabilityChain = ({
+  capability, capabilityChain, maxChainLength = constants.MAX_CHAIN_LENGTH
+}) => {
+  if(capabilityChain.length > maxChainLength) {
```

Comment:
`capabilityChain` does not include the given capability, this is the length of the `capabilityChain` array in the proof on the given capability.  Is the intent that the `maxChainLength` is inclusive of the given capability.  In which case this should be:
```
  if(capabilityChain.length + 1 > maxChainLength) {
```

Reply:
I think it should be the total number of zcaps, thus including the current capability, so the `+ 1` variant.